ports/espressif/boards/seeed_xiao_esp32_s3_sense/sdkconfig

```diff
@@ -0,0 +1,15 @@
+#
```

Comment:
Add new header.

Reply:
I'm checking the pinouts for the camera and will make some changes to the naming. There were two pins missing to be addressed and I will use the CAM_ prefix before the pin number. I will try to make some equivalence between the nomenclature used in Arduino.

Reply:
I think it would be best to make the pin names consistent with https://github.com/adafruit/circuitpython/blob/main/ports/espressif/boards/adafruit_esp32s3_camera/pins.c#L60-L73 (and maybe also include that data tuple in board)

Reply:
Done